dataflux_pytorch/benchmark/llama-70b/load.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+import torch
+
+from dataflux_pytorch import dataflux_checkpoint
+
+start_time = time.time()
+ckpt = dataflux_checkpoint.DatafluxCheckpoint(project_name="<PROJECT_NAME>",
+                                              bucket_name="<BUCKET_NAME")
```

Comment:
missing a bracket in this placeholder

Might be better to use flags or environment variables like our other demos/benchmarks

Reply:
+1